spec/models/embed/purl_spec.rb

```diff
@@ -123,6 +130,37 @@
     end
   end
 
+  describe 'all_files_downloadable' do
+    it 'returns true when all files are downloadable' do
+      stub_purl_response_with_fixture(multi_resource_multi_type_purl)
+      expect(described_class.new('12345').all_files_downloadable?).to be true
+    end
+
+    it 'returns false when one file is not downloadable' do
+      stub_purl_response_with_fixture(world_restricted_download_purl)
+      expect(described_class.new('12345').all_files_downloadable?).to be false
+    end
+  end
+
+  describe 'any_stanford_only_files' do
+    it 'returns true when one or more files are stanford only' do
```

Comment:
`when one or more files are stanford only` is a context so you should use the rspec `context` method to define those.

Reply:
Thanks for the tip.